src/stores/b2b-purchase-order-flow.md

```diff
@@ -0,0 +1,82 @@
+---
+b2b_only: true
+title: Purchase order workflow
+group: getting-started
+---
+
+Purchase orders allow companies to track and control spending. When purchase orders are enabled for a company account, all orders are automatically processed as purchase orders.
+
+The purchase order workflow can vary in a few ways:
```

Comment:
should a line item be added to mention how purchase order approvals work when purchase order rules is not enabled?

Reply:
Another excellent point.